src/whisk.rs

```diff
@@ -19,22 +20,25 @@ use crate::{
 
 pub const FIELD_ELEMENT_SIZE: usize = 32;
 pub const G1POINT_SIZE: usize = 48;
-pub const WHISK_SHUFFLE_PROOF_SIZE: usize = 4576;
+pub const WHISK_SHUFFLE_PROOF_SIZE: usize = 3136;
 // 48+48+32
 pub const TRACKER_PROOF_SIZE: usize = 128;
 
 // TODO: Customize
-const N: usize = 128;
+const N: usize = 16;
```

Comment:
Hmm this seems to interact with https://github.com/asn-d6/curdleproofs/pull/12 . Should I wait until https://github.com/asn-d6/curdleproofs/pull/12 gets merged, before merging this one?

Reply:
Sorry typo